tests/functional/ovndbcluster_controller_test.go

```diff
@@ -321,12 +321,13 @@ var _ = Describe("OVNDBCluster controller", func() {
 				g.Expect(k8sClient.Update(ctx, c)).Should(Succeed())
 			}).Should(Succeed())
 
-			//Check that error occurs
-			Eventually(func(g Gomega) {
-				conditions := GetOVNDBCluster(dbs[0]).Status.Conditions
-				cond := conditions.Get(condition.ExposeServiceReadyCondition)
-				g.Expect(cond.Status).To(Equal(corev1.ConditionFalse))
-			}).Should(Succeed())
```

Comment:
this error part is not what I got.

Reply:
that error was added on purpose some months back by @averdagu , let's check that first

Reply:
done